src/index.ts

```diff
@@ -1,4 +1,9 @@
+import { NativeModules } from 'react-native';
+
 import WebView from './WebView';
 
+export const RNCWebViewUtils = NativeModules.RNCWebViewUtils
+export const getWebViewDefaultUserAgent = RNCWebViewUtils.getWebViewDefaultUserAgent;
```

Comment:
We also don't need to export this, we need to use it only internally

Reply:
I'm not sure what the problem is then. I'll try to add a commit on top myself then for a review! )

Reply:
Done, PTAL